src/inc/quic_platform_winkernel.h

```diff
@@ -848,6 +848,16 @@ CxPlatRandom(
     _Out_writes_bytes_(BufferLen) void* Buffer
     );
 
+//
+// Process object abstraction
+//
+
+#define QUIC_PROCESS PEPROCESS
```

Comment:
Instead of putting `QUIC_OWNING_PROCESS=1` in the VS project files for kernel, just define it here directly. Then all kernel (i.e. datapath_winkernel.c) code can just assume it's set.